tests/components/unifi/test_device_tracker.py

```diff
@@ -962,29 +1077,36 @@ async def test_dont_track_devices(hass, aioclient_mock):
     await hass.async_block_till_done()
 
     assert len(hass.states.async_entity_ids(TRACKER_DOMAIN)) == 2
-
-    client_1 = hass.states.get("device_tracker.client_1")
-    assert client_1 is not None
-
-    device_1 = hass.states.get("device_tracker.device_1")
-    assert device_1 is not None
+    assert hass.states.get("device_tracker.client")
+    assert hass.states.get("device_tracker.device")
 
 
 async def test_dont_track_wired_clients(hass, aioclient_mock):
     """Test don't track wired clients config works."""
+    wireless_client = {
+        "essid": "ssid",
+        "hostname": "Wireless Client",
+        "is_wired": False,
+        "last_seen": 1562600145,
+        "mac": "00:00:00:00:00:01",
+    }
+    wired_client = {
```

Comment:
I'd break out the client dicts into pytest fixtures. They can be overridden or locally modified. All of the client details are probably not interesting for the developer for each test and the interesting items can be set explicitly in the test body, to make it clear what we want to test.

Reply:
I'll look into it, though I do like because its much quicker to get a grasp of what the test is about with everything local